Src/AutoFixture/PostprocessComposerExtensions.cs

```diff
@@ -0,0 +1,62 @@
+﻿using System;
+using System.Linq.Expressions;
+using AutoFixture.Dsl;
+
+namespace AutoFixture
+{
+    public static class PostprocessComposerExtensions
```

Comment:
This class contains extension methods for providing additional functionalities.

The most important one is the one that offers the possibility to accept a parameterless function as a parameter.

```csharp
b.With(p => p.Property, () => DoSomething());
```

Additionally, overloads with up to 4 arguments are provided.

```csharp
b.With<MyClass, MyPropertyType, string>(p => p.Property, (string a) => DoSomething(a));
b.With<MyClass, MyPropertyType, string, int>(p => p.Property, (string a, int b) => DoSomething(a, b));
b.With<MyClass, MyPropertyType, string, int. DateTime>(p => p.Property, (string a, int b, DateTime c) => DoSomething(a, b, c));
b.With<MyClass, MyPropertyType, string, int. DateTime, long>(p => p.Property, (string a, int b, DateTime c, long d) => DoSomething(a, b, c, d));
```